lib/minesweeper_2pl/app.rb

```diff
@@ -1,7 +1,41 @@
-module Minesweeper_2pl
+module Minesweeper
   class App
-    def call
-      "Hello World"
+    BOMB_PERCENT = 0.75
+    MAX_ROW_NUM = 20
+
+    attr_accessor :game, :cli
+
+    def initialize
+      cli = CLI.new
+      self.cli = cli
+      game_config = cli.start
+      game = Game.new(game_config.first, game_config.last)
+      self.game = game
+    end
+
+    def start
+      play_game
+      end_game
+    end
+
+    def play_game
```

Comment:
I think we've discussed this in person, but there might be another concept to extract and name here. Remember, abstract things like `Rules` and `GameLoop` can be classes/objects, too! What do you see here that could be extracted to a collaborator?